google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1230,16 +1230,34 @@ def file path,
         #   changed to a time in the future. If custom_time must be unset, you
         #   must either perform a rewrite operation, or upload the data again
         #   and create a new file.
+        # @param [Symbol, nil] checksum The type of checksum for the client to
+        #   automatically calculate and send with the create request to verify
+        #   the integrity of the object. If provided, Cloud Storage will only
+        #   create the file if the value calculated by the client matches the
+        #   value calculated by the service.
+        #
+        #   Acceptable values are:
+        #
+        #   * `md5` - Calculate and provide a checksum using the MD5 hash.
+        #   * `crc32c` - Calculate and provide a checksum using the CRC32c hash.
+        #   * `all` - Calculate and provide checksums for all available verifications.
```

Comment:
I'd include the colon to clarify that these are indeed symbols. e.g. ``` `:md5` ```

Reply:
I copied this existing style from `File#download`. We should figure out which style is the community standard and do a pass over all of the libraries.

Reply:
I'm surprised to see "all" here as we do not support multiple simultaneous checksum types in Python. Doesn't need attention, just mentioning it as a comment.